### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shanvijha30**
Hi @T8RIN !

I have added the Contributing guidelines as CONTRIBUTING.md file this documentation will contains set of guidelines that will help contributors during the contribution process.
File type markdown.

Thank you for considering my contribution!

### CODE REVIEW

**Comment by T8RIN** [comment]
And there’s not even a name for the repository :( Can you change it a bit, please?👀

**Comment by shanvijha30** [comment]
@T8RIN Can you please tell me where do I need to add the repository name?

Do you want me to add the repository name in Fork the repository?

**Comment by T8RIN** [comment]
Yup 🐸

**Comment by shanvijha30** [comment]
Okay will make the necessary changes

**Comment by shanvijha30** [comment]
@T8RIN I have the necessary changes if you could please review it!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- Fork this repository "ImageToolbox" by clicking on the "Fork" button. This will create a local copy of this respository on your GitHub profile.
+
+## 🌟 : Clone the forked repository
+
+- Once the repository is forked you need to clone it to your local machine.
+- Click on the "Code" button in the repository page and copy the link provided in the dropdown menu.
+
+
+```bash
+git clone https://github.com/<your-username>/<repo-name>
```
Comment (T8RIN) [inline CONTRIBUTING.md:24]: repo name

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+```bash
+git clone https://github.com/<your-username>/<repo-name>  
+```
+
+- Keep a reference to the original project in `upstream` remote.
+
+```bash  
+cd <repo-name>  
+git remote add upstream https://github.com/<upstream-owner>/<repo-name>
```
Comment (T8RIN) [inline CONTRIBUTING.md:31]: Repo name

**Review by T8RIN — CHANGES_REQUESTED** [review]

**Comment by shanvijha30** [comment]
@T8RIN I have made the changes please review it.

**Pull request merged**